tests/admin_views/test_nav_sidebar.py

```diff
@@ -42,7 +42,9 @@ def setUp(self):
 
     def test_sidebar_not_on_index(self):
         response = self.client.get(reverse("test_with_sidebar:index"))
-        self.assertContains(response, '<main class="main" id="main">')
+        self.assertContains(
+            response, '<main id="content-start" class="content" tabindex="-1">'
+        )
```

Comment:
I am actually not sure I remember the purpose of this line (and I think I might have written it) so I'm not sure if this should be testing the same (now div) element. That's probably more consistent with the existing test, anyway.

Reply:
To me it seems like the test is ensuring that the sidebar is not rendered on the index (either the admin home page, or a section, as opposed to a model-group). It does make more sense to test the div, which would be consistent with the original intention. I'll update it.